src/panels/config/zha/zha-node.ts

```diff
@@ -62,13 +55,31 @@ export class ZHANode extends LitElement {
       _showHelp: {},
       _selectedNodeIndex: {},
       _selectedNode: {},
+      _entities: {},
       _serviceData: {},
-      _selectedEntity: {},
+      _nodes: {},
     };
   }
 
+  public firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    if (this._nodes.length === 0) {
+      this._fetchDevices();
+    }
+    this.addEventListener("hass-service-called", (ev) =>
```

Comment:
@balloob I had to make a change here to clear selection and fetch fresh device list after remove is called. Just want to make sure the change is good before I merge this and the backend PR.

Reply:
If a remove is successful, just remove the device on the client side instead of refreshing devices?

Reply:
I can do that. Thx

Reply:
Going to merge this now so I can do a build, you can do it in another PR.